rudder-web/src/main/scala/com/normation/rudder/web/services/ReportDisplayer.scala

```diff
@@ -102,6 +102,9 @@ class ReportDisplayer(
     val id = JsNodeId(node.id)
     val callback =  SHtml.ajaxInvoke(() => SetHtml("reportsDetails",displayReports(node)) )
     Script(OnLoad(JsRaw(s"""
+      if($$("[aria-controls='node_reports']").hasClass('ui-tabs-active')){
```

Comment:
Maybe something can be done at another level (DisplayNode, ShowNodeDetailsFromNode)